cli/src/commands/branches/create.ts

```diff
@@ -57,12 +66,12 @@ export default class BranchesCreate extends BaseCommand {
           throw err;
         }
       }
-      // TODO calculate associated git branch with `from` xata branch
-      const gitBase = from || defaultGitBranch();
-      createBranch(branch, gitBase);
     }
 
     const result = await xata.branches.createBranch(workspace, database, branch, from);
+    if (useGit) {
+      await xata.databases.addGitBranchesEntry(workspace, database, { xataBranch: branch, gitBranch: branch });
```

Comment:
I think we can omit this step since the git branch and the xata branch have the same name.